Responsive Ecommerce Landing Page/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecommerce Website</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="image/icon.png" type="image/x-icon">
+</head>
+
+<body>
+    <div id="wrapper">
+
+        <!-- Navbar -->
+        <nav>
+            <div class="nav-left">
+                <a href="">KS.</a>
+            </div>
+            <div class="nav-mid">
+                <a href="#Home">Home</a>
```

Comment:
this should link to the repo landing page /index.html then add your information in the repos index.html so your project and github can be view from the repo landing page

Reply:
line 21 should be <a href="/index.html">Home</a> for ease of navigation

Reply:
also in the root directory add your information so that your site is visible on the home page